Engine/ScriptAnalyzer.cs

```diff
@@ -283,20 +283,19 @@ internal bool ParseProfile(object profileObject, PathIntrinsics path, IOutputWri
             Debug.Assert(includeRuleList != null);
             Debug.Assert(excludeRuleList != null);
 
-            switch (key.ToLower())
+            if (key.Equals("severity", StringComparison.OrdinalIgnoreCase))
             {
-                case "severity":
-                    severityList.AddRange(values);
-                    break;
-                case "includerules":
-                    includeRuleList.AddRange(values);
-                    break;
-                case "excluderules":
-                    excludeRuleList.AddRange(values);
-                    break;
-                default:
-                    return false;
+                severityList.AddRange(values);
```

Comment:
It strikes me that it would be simpler to just `return true` here and in the next clause and just `return false` at the end by default

Reply:
yes, I agree.